docs/source/conf.py

```diff
@@ -57,7 +57,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['.docs/source/notebooks/community_tutorials/*']
+exclude_patterns = ['.docs/source/notebooks/community_tutorials/*', '.docs/source/notebooks/X_dumping_data.ipynb']
```

Comment:
This line stays or is removed depending on whether we keep/remove the notebook from the merge